src/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.AddSub.cs

```diff
@@ -5,6 +5,68 @@ namespace System.Numerics
 {
     internal static partial class BigIntegerCalculator
     {
+        public static uint[] Add(uint left, uint right)
+        {
+            // Executes the trivial addition for two 32-bit integers.
+            // Since the caller is working with uint[] objects, we're
+            // doing him a favor with already providing it that way.
+
+            uint[] bits = new uint[2];
```

Comment:
I only try to keep this simple and thought an additional compression before the actual compression doesn't really help. Since (nearly) every other operation (within `BigIntegerCalculator` and `BigIntegerBuilder`) allocates the space *probably* needed (it's not always that easy to know this beforehand), I haven't seen any point to spare a few bits.

I'll make a performance comparison (for the trivial addition and multiplication), and update the code/comments accordingly.